packages/typespec-autorest/test/formats.test.ts

```diff
@@ -71,4 +71,48 @@ describe("typespec-autorest: format", () => {
     );
     expectDiagnosticEmpty(diagnostics);
   });
+
+  it("ensures certain scalars emit int32 and int64 formats", async () => {
+    const [res, _] = await emitOpenApiWithDiagnostics(
+      `
+      @service
+      namespace Test;
+
+      model Widget {
+        intA: int32;
+        intB: int64;
+        intC: safeint;
+        intD: numeric;
+        intE: integer;
+      }
+      `
+    );
+    const model = res.definitions!["Widget"]!;
+    deepStrictEqual(model, {
+      properties: {
+        intA: {
+          format: "int32",
+          type: "integer",
+        },
+        intB: {
+          format: "int64",
+          type: "integer",
+        },
+        intC: {
+          format: "int64",
+          type: "integer",
+        },
+        intD: {
```

Comment:
This emits an integer?

Reply:
It does. https://github.com/Azure/typespec-azure/blob/main/packages/typespec-autorest/src/openapi.ts#L2149

Reply:
That being said, this test has been removed. There is already a test in autorest.